rpcs3/rpcs3qt/user_create_dialog.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "stdafx.h"
+
+#include <QDialog>
+#include <QPushButton>
+#include <QLineEdit>
+#include <QHBoxLayout>
+
+class user_create_dialog : public QDialog
+{
+	Q_OBJECT
+public:
+	explicit user_create_dialog(const std::string& dir = "", QWidget* parent = nullptr);
+	QString GetText();
+	QPushButton* GetOkButton();
```

Comment:
replace GetOkButton with a Q_SIGNAL.
then use &user_create_dialog::<yourSignal> instead of &QAbstractButton::clicked in the user_manager_dialog

Reply:
Yeah, I had a feeling it was odd to pass the button object around. So then I attach the signal to the ok button click in user_create_dialog?